runtime/src/main/java/org/corfudb/util/serializer/Serializers.java

```diff
@@ -38,10 +38,19 @@
      */
     public static ISerializer getSerializer(Byte type) {
         if (type <= SYSTEM_SERIALIZERS_COUNT) {
-            return serializersMap.get(type);
+            if (serializersMap.containsKey(type)) {
+                return serializersMap.get(type);
+            }
         } else {
-            return customSerializers.get(type);
+            if (customSerializers.containsKey(type)) {
+                return customSerializers.get(type);
+            }
         }
+
+        String message = String.format("Serializer with type code %s not found. " +
+                "Please check custom serializers for this client.", type.intValue());
+        log.error(message);
```

Comment:
Can you put this message to log directly?

Reply:
Done.